### ISSUE TITLE
Duplicate class

### ISSUE THREAD

**Issue by pokerfaceCmy** [braintree/braintree_android#436]
My build.gradle(:app)
```
    implementation('com.braintreepayments.api:drop-in:5.2.2'){
//        {
//            exclude group:'com.braintreepayments.api',module:'data-collector'
//            exclude group:'com.braintreepayments.api',module:'braintree'
//            exclude group:'com.braintreepayments.api',module:'braintree-core'
//            exclude group:'com.braintreepayments.api',module:'shared-utils'
//        }
    }

    implementation ('com.braintreepayments.api:paypal:4.4.1')

```
My AndroidManifest.xml
```
        <activity android:name="com.wetech.aus.tennis.app.domain.club.BraintreeActivity"
            android:launchMode="singleTask">
            <intent-filter>
                <action android:name="android.intent.action.VIEW" />
                <category android:name="android.intent.category.DEFAULT" />
                <category android:name="android.intent.category.BROWSABLE" />
                <data android:scheme="${applicationId}.braintree"/>
            </intent-filter>
        </activity>
```

```
Duplicate class com.braintreepayments.api.BuildConfig found in modules jetified-braintree-3.17.4-runtime (com.braintreepayments.api:braintree:3.17.4) and jetified-braintree-core-4.4.1-runtime (com.braintreepayments.api:braintree-core:4.4.1)
Duplicate class com.braintreepayments.api.CrashReporter found in modules jetified-braintree-3.17.4-runtime (com.braintreepayments.api:braintree:3.17.4) and jetified-braintree-core-4.4.1-runtime (com.braintreepayments.api:braintree-core:4.4.1)
Duplicate class com.braintreepayments.api.DataCollector found in modules jetified-braintree-3.17.4-runtime (com.braintreepayments.api:braintree:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class com.braintreepayments.api.DataCollector$1 found in modules jetified-braintree-3.17.4-runtime (com.braintreepayments.api:braintree:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class com.braintreepayments.api.DataCollector$1$1 found in modules jetified-braintree-3.17.4-runtime (com.braintreepayments.api:braintree:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class com.braintreepayments.api.Json found in modules jetified-core-3.17.4-runtime (com.braintreepayments.api:core:3.17.4) and jetified-shared-utils-4.4.1-runtime (com.braintreepayments.api:shared-utils:4.4.1)
Duplicate class com.braintreepayments.api.PaymentMethod found in modules jetified-braintree-3.17.4-runtime (com.braintreepayments.api:braintree:3.17.4) and jetified-braintree-core-4.4.1-runtime (com.braintreepayments.api:braintree-core:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.Environment found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.InvalidInputException found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.MagnesResult found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.MagnesSDK found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.MagnesSettings found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.MagnesSettings$1 found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.MagnesSettings$Builder found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.MagnesSource found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.a.a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.a.b found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.a.c found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.a.d found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.b found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.b.a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.b.a$a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$b found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$b$a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$b$b found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$c found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$d found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$d$a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$d$b found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$d$c found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$d$d found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$d$e found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$d$f found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$e found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$e$a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$e$b found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$e$c found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$f found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$f$a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$g found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$h found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$h$a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$h$b found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$h$c found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$h$d found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$i found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$j found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$k found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$k$a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$k$b found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$l found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$m found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.c$n found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.d found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.e found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.f found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.f$1 found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.g found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.g$a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.g$b found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.g$c found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.g$d found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.g$e found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.h found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.h$1 found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.h$2 found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.h$a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.h$b found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.i found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.j found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.j$1 found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.k found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.k$1 found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.k$2 found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.k$a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.k$b found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.l found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.l$1 found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.l$a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.m found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.a$1 found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.b found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.b$1 found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.base.MagnesNetworking found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.base.MagnesNetworkingFactory found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.base.MagnesNetworkingFactoryImpl found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.base.MagnesNetworkingFactoryImpl$1 found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.base.PayPalCertificate found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.base.a found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.base.b found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.base.c found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.base.d found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.base.e found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.base.e$1 found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)
Duplicate class lib.android.paypal.com.magnessdk.network.base.f found in modules jetified-data-collector-3.17.4-runtime (com.paypal.android.sdk:data-collector:3.17.4) and jetified-data-collector-4.4.1-runtime (com.braintreepayments.api:data-collector:4.4.1)

Go to the documentation to learn how to Fix dependency resolution errors.
```

**Comment by gauranggoda** [comment]
I am facing a similar issue while using the below dependency. did anyone found a solution to this?
```
implementation 'com.braintreepayments.api:drop-in:5.2.2'

//braintree - local payments
implementation 'com.braintreepayments.api:local-payment:4.6.0'
```

**Comment by sarahkoop** [comment]
Hi @gauranggoda `drop-in` v5 is not compatible with v4 of the Braintree core SDK modules. We have released v6.0.0-beta1 of `drop-in` that is compatible.
